src/notes/FillPlaceholder.jsx

```diff
@@ -67,6 +67,38 @@ export default class FillPlaceholder extends Component {
         document.removeEventListener('mousedown', this.handleClick, false);
     }
 
+    fillFromData = (data) => {
+        const { placeholder } = this.props;
+        let errorToReturn = null, error; // return first error only
+
+        let entryIndex = 0;
+        if (placeholder.multiplicity === 'many') {
+            let lastEntryIndex = placeholder.entryShortcuts.length - 1;
+            const valFirstField = placeholder.getAttributeValue(data.fields[0].name, lastEntryIndex);
```

Comment:
[DL1]  The logic here is looking at the first field of the last entry to determine if it should add a new entry or fill out the existing entry.  Should it look at all the fields and see if any of them are already filled out?